### ISSUE TITLE
TypeError after updating to 0.1.0

### ISSUE THREAD

**Issue by larslaade** [SirPepe/ornament#5]
When I update to from 0.0.27 to 0.1.0 I get the following errors:

with Firefox: "Uncaught TypeError: context.metadata is undefined"

```
@attr(json(), {
    reflective: false
  }) accessor rate: Rate = {
    months: 0,
    amount: 0,
  }
```

with Chrome: "Uncaught TypeError: Cannot read properties of undefined (reading 'Symbol()')"

```
@define('my-component')
export class MyComponent extends HTMLElement {
 // [...]
}
```

with Safari: "TypeError: undefined is not an object (evaluating 'context.metadata[META]')"

Build with webpack + ts-loader + TS (all newest versions), no other fancy stuff. Did I miss a config?

**Comment by SirPepe** [comment]
Sounds serious. Can you paste the contents all your relevant config files here?

**Comment by larslaade** [comment]
Created an example in a repository [https://github.com/larslaade/ornament-example](https://github.com/larslaade/ornament-example)

inital commit with 0.0.27
recent commit with 0.1.0

**Comment by SirPepe** [comment]
Ok, here is what's going on:

- Ornament 0.1.0 started using [decorator metadata](https://github.com/tc39/proposal-decorator-metadata) for, well, metadata
- TS 5.2 does support decorator metadata, but does not include a runtime polyfill by default. This information is buried in the last paragraphs of the [release announcement](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-2.html#decorator-metadata)
- This, along with the fact that [decorator metadata is broken in a different way in babel](https://github.com/SirPepe/ornament/issues/6), will be dealt with in 0.1.1, when ornament will go back to using weak maps for metadata

For the time being, you could add the polyfills that the TS release announement talks about. Or just wait for a few hours until I have stable internet access and can publish the new version :upside_down_face:

**Comment by SirPepe** [comment]
Fixed in 0.1.1
